tests/smtp-long-DATA-line-03-frames-ips/suricata.yaml

```diff
@@ -0,0 +1,24 @@
+%YAML 1.1
```

Comment:
This is the same pcap that can be deduplicated, right ?

Reply:
done now